components/script/dom/svgsvgelement.rs

```diff
@@ -60,15 +88,32 @@ impl LayoutSVGSVGElementHelpers for LayoutDom<SVGSVGElement> {
         unsafe {
             let SVG = &*self.unsafe_get();
 
+            let ctx = SVG.get_or_init_svg_context();
```

Comment:
This should be called from the constructor of SVGSVGElement instead. The operations in get_or_init_svg_context() can't be used from the layout thread; they're only valid in the script thread.